Tests/lib/Initialize-Test.ps1

```diff
@@ -9,6 +9,18 @@ if (!(Get-Module | Where-Object { $_.Name -eq 'Microsoft.PowerShell.Management'
 }
 
 Import-Module $PSScriptRoot\TestHelpersModule -Verbose:$false
+Register-Mock Import-Module
+function Get-LocalizedString {
```

Comment:
This cmdlet is so widely used that I created a mock implementation here so each test doesn't need to mock it out.